Tests/ArmoniK.EndToEndTests/ArmoniK.EndToEndTests.Common/ArmoniK.EndToEndTests.Common.csproj

```diff
@@ -13,7 +13,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="ArmoniK.Api.Common" Version="3.6.1" />
+    <PackageReference Include="ArmoniK.Api.Common" Version="3.8.1" />
```

Comment:
be sure you will add feat: in the PR of Ext.C# !!

Reply:
Already done